### ISSUE TITLE
System Notifications should automatically disappear after the end date

### ISSUE THREAD

**Issue by krusche** [ls1intum/Artemis#1328]
System notifications have an end date, but they do not automatically disappear.
Users have to reload the Artemis client application in the browser.

It would be better if they disappear automatically.
This should also respect updates of the system notifications (e.g. changing the start / end date)

**Comment by sascha11110** [comment]
At least the deletion of system notifications is fixed here: https://github.com/ls1intum/Artemis/pull/1599

**Comment by Malyuk-A** [comment]
Issue Update:
I know I can fix this issue but the system notifications have multiple issues (which can also be seen by looking at multiple other related open GitHub Issues). It feels wrong to fix a bit of a more or less broken system. The core of the system notifications has critical design issues. I studied the code and it was created with only one active notification in mind (not multiple). (This is also already a GitHub Issue).
To properly solve these issues the system notifications have to undergo a big full-stack refactoring.
I closed my PR because I currently have multiple other features and issues to solve and simply have no time to spend on a major refactoring.
